src/editor/Cloud.tsx

```diff
@@ -0,0 +1,49 @@
+import toast from "react-hot-toast"
+
+import { toGateway } from "~/lib/ipfs-parser"
+import { UploadFile } from "~/lib/upload-file"
+
+import { ICommand } from "."
+
+const uploadResources = async (text: string) => {
+  const markdownImageRegex = /!\[(.*)\]\((.+)\)/
+  const match = text.match(markdownImageRegex)
+  if (match) {
+    const altText = match[1]
+    const url = match[2]
+    console.log({ url })
+    const blob = await fetch(url, {
+      mode: "no-cors",
+    }).then((response) => response.blob())
+    const key = (await UploadFile(blob)).key
+    const newUrl = toGateway(key)
```

Comment:
It is better to change it to the format of `ipfs://{cid}` instead of hardcoding the gateway here.

Reply:
Resolved. Unified url to `ipfs://{cid}`.